### PR TITLE
Update Layouts & Keymaps

### PR DESCRIPTION
**Pull request by mrsendyyk**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ai03/soyuz/keymaps/mrsendyyk/keymap.c [hunk 1]
```diff
@@ -15,16 +15,16 @@
  * along with this program. If not, see <https://www.gnu.org/licenses/>.
```
Comment (zvecr) [inline keyboards/ai03/soyuz/keymaps/mrsendyyk/keymap.c:15]: Why move this from the community folder, that seems like taking a step backwards.

File: keyboards/ai03/soyuz/keymaps/via/keymap.c [hunk 2]
```diff
@@ -13,19 +14,21 @@
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <https://www.gnu.org/licenses/>.
  */
- 
+
+// ai03 Soyuz VIA Keymap by Sendy YK <https://mr.sendyyk.com>.
+
```
Comment (zvecr) [inline keyboards/ai03/soyuz/keymaps/via/keymap.c:19]:
```suggestion

```

File: keyboards/ai03/soyuz/keymaps/via/keymap.c [hunk 3]
```diff
@@ -13,19 +14,21 @@
...
 #include QMK_KEYBOARD_H
 
 enum layer_names {
-    _BASE, // Default Layer
+    _MAIN, // Main Layer
```
Comment (zvecr) [inline keyboards/ai03/soyuz/keymaps/via/keymap.c:23]:
```suggestion
    _BASE, // Default Layer
```

File: keyboards/ai03/soyuz/keymaps/via/keymap.c [hunk 4]
```diff
@@ -13,19 +14,21 @@
...
     _FN, // Fn Layer 1
     _FN2, // Fn Layer 2
     _FN3 // Fn Layer 3
 };
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
-	// Default Layer
-    [_BASE] = LAYOUT_ortho_5x4(
+	// Main Layer
+    [_MAIN] = LAYOUT_ortho_5x4(
```
Comment (zvecr) [inline keyboards/ai03/soyuz/keymaps/via/keymap.c:31]:
```suggestion
	// Default Layer
    [_BASE] = LAYOUT_ortho_5x4(
```

File: keyboards/dz60/keymaps/mrsendyyk/keymap.c [hunk 5]
```diff
@@ -16,43 +16,34 @@
  */
```
Comment (zvecr) [inline keyboards/dz60/keymaps/mrsendyyk/keymap.c:16]: Same comment as the numpad_5x4 change.

File: layouts/default/60_ansi_arrow/default_60_ansi_arrow/keymap.c [hunk 6]
```diff
@@ -15,39 +15,30 @@
  * along with this program. If not, see <https://www.gnu.org/licenses/>.
```
Comment (zvecr) [inline layouts/default/60_ansi_arrow/default_60_ansi_arrow/keymap.c:15]: Was there a reason to update this default community layout?

**Review by zvecr — CHANGES_REQUESTED** [review]
Not sure i see the point in some of the changes, it would be nice to see the "why" behind some of them as we can already see the changes via the files changed tab.

File: keyboards/ai03/soyuz/keymaps/mrsendyyk/keymap.c (hunk 1 above)
Comment (mrsendyyk) [reply keyboards/ai03/soyuz/keymaps/mrsendyyk/keymap.c:15]: Based on @noroadsleft comment on [#10338](https://github.com/qmk/qmk_firmware/pull/10338#discussion_r492310612), matrix wiring for the board being used with the keymap are accurate for the Soyuz, but some boards with this layout wire things differently. I'm afraid in the future if i have another numpad 5x4 pcb again, it won't be work. CMIIW @zvecr.

Comment (zvecr) [reply keyboards/ai03/soyuz/keymaps/mrsendyyk/keymap.c:15]: Matrix wiring doesn't matter, that's the whole point of abstraction within layout macros. Anything that supports `numpad_5x4` will work with the keymap.

File: keyboards/dz60/keymaps/mrsendyyk/keymap.c (hunk 5 above)
Comment (mrsendyyk) [reply keyboards/dz60/keymaps/mrsendyyk/keymap.c:16]: Done

File: layouts/default/60_ansi_arrow/default_60_ansi_arrow/keymap.c (hunk 6 above)
Comment (mrsendyyk) [reply layouts/default/60_ansi_arrow/default_60_ansi_arrow/keymap.c:15]: Just a little updates

File: keyboards/ai03/soyuz/keymaps/mrsendyyk/keymap.c (hunk 1 above)
Comment (mrsendyyk) [reply keyboards/ai03/soyuz/keymaps/mrsendyyk/keymap.c:15]: Oh ok, then i will move again to the community folder.

**Pull request closed without merging**